tests/test_custom_models.py

```diff
@@ -874,7 +887,8 @@ def test_only_params_are_updated(self, test_name, model_id, config_cls, config_k
         model_before = copy.deepcopy(model)
 
         model.train()
-        optimizer = torch.optim.SGD(model.parameters(), lr=0.5)
+        lr = 100.0 if config_kwargs.get("use_dora") else 0.5
```

Comment:
Wow, is 100 really necessary? For me, even 1.0 seems to work (tested 10x on CPU and GPU).

Reply:
Oh yeah!

I ran the `1.0` experiment 1000x on CPU and GPU, and it did not go well. I tried increasing the lr in the following manner 1 -> 2 -> 4 -> 10 -> 50 -> 100. 100 was the only lr that passed for 1000x tests.

Reply:
This is only required for dora + embedding, right? So let's adjust the check, same as below. And let's add a comment that this high learning rate was found through testing to be necessary to avoid flakiness.

Reply:
Updated!